Include the new query service streaming updater (#150)
wmde/wikidata-wikibase-architecture: Glossary.md

[Gadgets](https://www.mediawiki.org/wiki/Extension:Gadgets) are made up of JavaScript and/or CSS Snippets located on pages in the MediaWiki namespace. These snippets add functionality to the wiki itself and can be enabled and disabled via preferences.

## [Hadoop](https://hadoop.apache.org/)

> Allows for the distributed processing of large data sets across clusters of computers using simple programming models.

Reviewer: ```suggestion
Apache Hadoop is a storage system with a collection of libraries, that allow for the distributed processing of large data sets across clusters of computers using simple programming models.
```

The terms "Wikibase Item" and "Wikidata Item" are used interchangeably throughout the documentation. They are the same thing that lives in different contexts - Wikidata or another WikibaseRepo instance.

## [Kafka](https://kafka.apache.org/intro)

> Kafka is an event streaming platform.

Reviewer: ```suggestion
Apache Kafka is an event streaming platform.
```

Properties are [Entities](#entity) that describe a relationship between [Items](#item) (or other Entities) and [values](#datavalue) of the Property. Typical Properties are _population_ (using numbers as values), _binomial name_ (using strings as values), but also _has father_ and _author of_ (both using Items as values).

## [QueryBuilder](https://www.wikidata.org/wiki/Wikidata:Query_Builder)

> The Query Builder provides a visual interface for building a simple query.

Reviewer: ```suggestion
The Query Builder provides a visual interface for building simple Wikibase data queries.
```

A textual form of RDF, common in Wikibase is [Turtle](#turtle).

## [Recent changes](https://www.mediawiki.org/wiki/Help:Recent_changes)

> A feature withing [MediaWiki](#mediawiki) exposed via API and UI that lists recent changes in reverse order, with change date and time, change in the page size, user who created the page, and edit summary.

Reviewer: ```suggestion
A feature within [MediaWiki](#mediawiki) exposed via API and UI that lists recent changes in reverse order, with change date and time, change in the page size, user who created the page, and edit summary.
```

Wikibase is a collection of software and services to enable creation and maintenance of a powerful, flexible and customizable knowledge base.
[Wikidata](#wikidata) is the largest current public deployement of this system.

## Wikibase as a Service

Wikibase as a Service is a platform allowing easy creation of multiple Wikibase instances using a single shared set of resources.

> Primarily developed under the name WBStack at <https://wbstack.com> at the start of 2022 a service was created at <https://wikibase.cloud>

Reviewer: ```suggestion
Primarily developed under the name WBStack at <https://wbstack.com>, at the start of 2022 a service was created at <https://wikibase.cloud>
```
